packages/react-components/stories/header-bar.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { HeaderBar } from '../lib/header-bar';
+import { createStyles, IconButton, makeStyles, Toolbar, Typography } from '@material-ui/core';
+import Tab from '@material-ui/core/Tab';
+import TabPanel from '@material-ui/lab/TabPanel';
+import TabContext from '@material-ui/lab/TabContext';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import { NavigationBar } from '../lib/navigation-bar';
+import { BannerTab } from '../lib/banner-tab';
+
+export default {
+  title: 'Header Bar',
+  component: HeaderBar,
+} as Meta;
+
+export const NavBar: Story = () => {
+  const [value, setValue] = React.useState('building');
+
+  const onTabChange = (event: React.ChangeEvent<unknown>, newValue: string) => {
+    setValue(newValue);
+  };
+
+  return (
+    <>
+      <TabContext value={value}>
```

Comment:
Not exactly sure how this works, but I don't think we should be using the TabContext ourselves, the new `NavigationBar` already contains a tab bar and iirc it will provide it's own TabContext.